CQCategory.m

```diff
@@ -0,0 +1,21 @@
+//
+//  CQCategory.m
+//  OptionSelector
+//
+//  Created by Lauren Caponong on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CQCategory.h"
+
+@implementation CQCategory
+
+- (void)initializeData {
```

Comment:
Why is this method necessary? Since it doesn't actually affect the state of the code. :D